sormas-app/app/src/main/java/de/symeda/sormas/app/backend/common/DatabaseHelper.java

```diff
@@ -1594,6 +1594,10 @@ public void onUpgrade(SQLiteDatabase db, ConnectionSource connectionSource, int
 				currentVersion = 228;
 				getDao(Person.class).executeRaw("ALTER TABLE person ADD COLUMN externalId varchar(255);");
 
+			case 229:
+				currentVersion = 229;
+				getDao(Visit.class).executeRaw("ALTER TABLE visits ADD COLUMN origin varchar(255);");
```

Comment:
Set value to USER for existing visits.  Also make sure the the API minimum required version is updated (see https://github.com/hzi-braunschweig/SORMAS-Project/blob/development/GUIDE_ADD_NEW_FIELD.md#i-adding-the-field-to-the-sormas-api point 6)